nginx-1.21.4-wolfssl.patch

```diff
@@ -150,7 +266,7 @@ index 84afecd0..fe7e328e 100644
          c->ssl->session = sess;
  
          c->ssl->save_session(c);
-@@ -1674,7 +1708,9 @@ ngx_ssl_get_session(ngx_connection_t *c)
+@@ -1674,7 +1747,9 @@
```

Comment:
What are the below patches? Maybe I need to pull this down and review and patch to see. Is this white space?

Reply:
That whitespace is created by `diff` to keep alignment correct.